akka-stream/src/main/scala/akka/stream/io/SslTls.scala

```diff
@@ -138,14 +141,14 @@ object SslTls {
  * unwrapping [[SendBytes]].
  */
 object SslTlsPlacebo {
-  val forScala = scaladsl.BidiFlow() { implicit b ⇒
+  val forScala: SslTls.ScalaFlow = scaladsl.BidiFlow() { implicit b ⇒
```

Comment:
Yes, the type is now wider, which I'd consider to be better.
Which test compilation failure do you mean?